src/main/java/com/conveyal/gtfs/error/OverlappingTripsInBlockError.java

```diff
@@ -12,16 +12,16 @@
     public static final long serialVersionUID = 1L;
 
     public final String[] tripIds;
-    public final String affectedEntityId;
     public final Priority priority = Priority.HIGH;
+    public final String routeId;
 
-    public OverlappingTripsInBlockError(long line, String field, String affectedEntityId, Route route, String[] tripIds, Priority priority) {
-        super("trips", line, field);
+    public OverlappingTripsInBlockError(long line, String field, String affectedEntityId, String routeId, String[] tripIds) {
```

Comment:
Right, I'm thinking that it should be the `block_id`.  I'll change that param name.